server/vertx-server/src/main/scala/sttp/tapir/server/vertx/decoders/VertxRequestBody.scala

```diff
@@ -85,7 +85,9 @@ class VertxRequestBody[F[_], S <: Streams[S]](
             .map { fu =>
               mp.partType(fu.name())
                 .flatMap(rawBodyType => extractFilePart(fu, rawBodyType))
-                .map(body => Part(fu.name(), body, fileName = Option(fu.fileName())))
+                .map(body =>
+                  Part(fu.name(), body, contentType = MediaType.parse(fu.contentType()).toOption, fileName = Option(fu.fileName()))
```

Comment:
The `@Nullable` annotation is not given to the `contentType` method of the FileUpload interface.
Also, in the processing of `io.netty.handler.codec.http.multipart.HostPostMultipartRequestDecoder`, the default `application/octet- stream` is set if the ContentType is not specified.
For this reason, I assumed that at least NULL was not set when this processing block was reached.

Reply:
No, I think that's fine, esp that tests pass